examples/StandAloneFileWatcher/settings.py

```diff
@@ -33,19 +33,19 @@
     'port': 22916,
 
     # public and secret key for the standalonelistener agent.
-    # These can be created using the command:  volttron-ctl auth keypair
+    # These can be created using the command:  vctl auth keypair
```

Comment:
just an fyi vctl is an alias for volttron-ctl  though much easier....I don't know if you knew that or not

Reply:
I did know that thanks, thought I would just change this to use the more common version of the command (at least that's the one I have seen more often)